rllib/agents/sac/sac_model.py

```diff
@@ -121,44 +115,51 @@ def build_q_net(name, observations, actions):
         else:
             self.twin_q_net = None
 
-        self.log_alpha = tf.Variable(0.0, dtype=tf.float32, name="log_alpha")
+        self.log_alpha = tf.Variable(
+            np.log(initial_alpha), dtype=tf.float32, name="log_alpha")
         self.alpha = tf.exp(self.log_alpha)
 
         self.register_variables([self.log_alpha])
 
-    def get_q_values(self, model_out, actions):
+    def get_q_values(self, model_out, actions=None):
         """Return the Q estimates for the most recent forward pass.
 
         This implements Q(s, a).
 
         Arguments:
             model_out (Tensor): obs embeddings from the model layers, of shape
                 [BATCH_SIZE, num_outputs].
-            actions (Tensor): action values that correspond with the most
-                recent batch of observations passed through forward(), of shape
-                [BATCH_SIZE, action_dim].
+            actions (Optional[Tensor]): Actions to return the Q-values for.
+                Shape: [BATCH_SIZE, action_dim]. If None (discrete action
+                case), return Q-values for all actions.
 
         Returns:
             tensor of shape [BATCH_SIZE].
         """
-        return self.q_net([model_out, actions])
+        if actions is not None:
```

Comment:
Should the api for discrete and continuous api here be the same?

Reply:
For discrete, we don't have to pass any actions in. We get the probs for all actions at the same time for a given observation.